Makefile

```diff
@@ -23,3 +23,34 @@ test-e2e: GO_TEST_PACKAGES :=./test/e2e/...
 test-e2e: GO_TEST_FLAGS += -v -count=1
 test-e2e: test-unit
 .PHONY: test-e2e
+
+# -------------------------------------------------------------------
+# OpenShift Tests Extension (Cluster OpenShift Controller Manager Operator)
+# -------------------------------------------------------------------
+TESTS_EXT_BINARY := cluster-openshift-controller-manager-operator-tests-ext
+TESTS_EXT_PACKAGE := ./cmd/cluster-openshift-controller-manager-operator-tests-ext
+
+TESTS_EXT_GIT_COMMIT := $(shell git rev-parse --short HEAD)
+TESTS_EXT_BUILD_DATE := $(shell date -u +%Y-%m-%dT%H:%M:%SZ)
+TESTS_EXT_GIT_TREE_STATE := $(shell if git diff-index --quiet HEAD --; then echo clean; else echo dirty; fi)
+
+TESTS_EXT_LDFLAGS := \
+	-X 'main.CommitFromGit=$(TESTS_EXT_GIT_COMMIT)' \
+	-X 'main.BuildDate=$(TESTS_EXT_BUILD_DATE)' \
+	-X 'main.GitTreeState=$(TESTS_EXT_GIT_TREE_STATE)'
+
+.PHONY: tests-ext-build
+tests-ext-build:
+	GOOS=$(if $(GOOS),$(GOOS),$(shell go env GOOS)) \
```

Comment:
Rerfer to https://github.com/openshift/cluster-kube-storage-version-migrator-operator/pull/130